src/backend/src/form-responses/dto/create-form-response.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class CreateFormResponseDto {
+  readonly answer: string;
```

Comment:
Should a form response also contain the id of the question it's answering?

Reply:
also are we using these at all yet or are they just here?

Reply:
yeah I feel like we can just remove the single form response DTO. But yes, we are using the DTOs.

Reply:
wouldn't it make more sense to remove the multi form response, and then just refer to a multi form response as an array of singular form responses?

Reply:
I see. For multi form response, we only need to query userId once, and save all updated formResponse once. If we use a for loop with single form response, we need to kinda call findOne a bunch of times.